backend/de.metas.adempiere.adempiere/client/src/main/java-legacy/org/compiere/apps/Attachment.java

```diff
@@ -213,10 +213,10 @@ public void dispose()
 	 */
 	private void loadAttachments()
 	{
-		final AttachmentEntryService attachmentEntryService = SpringContextHolder.instance.getBean(AttachmentEntryService.class);
+		final AttachmentService attachmentService = SpringContextHolder.instance.getBean(AttachmentService.class);
```

Comment:
IMHO u can consider dropping Attachment class and code around.
on the method that opens the attachment dialog u can throw unsupported operation exception.